### PR TITLE
build: Enable jscs jsDoc rule 'checkAnnotations' and make pass

### PR DESCRIPTION
**Pull request by ricordisamoa**
Allow 'licence' tag, fix 3 'returns' that should have been 'return'.

### CODE REVIEW

File: .jscsrc [hunk 1]
```diff
@@ -1,4 +1,11 @@
 {
 	"preset": "wikimedia",
-	"jsDoc": null
+	"jsDoc": {
+		"checkAnnotations": {
+			"preset": "jsduck5",
+			"extra": {
+				"licence": "some"
```
Comment (Nikerabbit) [inline .jscsrc]: I thought the normal fix is to change header comments to use /*!

**Comment by Nikerabbit** [comment]
@ricordisamoa ping?

File: .jscsrc (hunk 1 above)
Comment (ricordisamoa) [inline .jscsrc]: I'm not very familiar with that style but I'll change it.

**Comment by ricordisamoa** [comment]
@Nikerabbit done

**Pull request merged**